src/t_stream.c

```diff
@@ -1350,6 +1367,155 @@ robj *createObjectFromStreamID(streamID *id) {
                         id->ms,id->seq));
 }
 
+/* Returns non-zero if the ID is 0-0. */
+int streamIDEqZero(streamID *id) {
+    return !(id->ms || id->seq);
+}
+
+/* A helper that returns non-zero if the range from 'start' to the stream's
+ * end doesn't contain a tombstone.
+ *
+ * NOTE: this assumes that the caller had verified that 'start' is less than
+ * 's->last_id'. */
+int streamIsContiguousRange(stream *s, streamID *start) {
```

Comment:
I feel that `streamIsContiguousRange` does not reflect the true usefulness of this method, perhaps `streamIsContainTombstoneRange` is more appropriate, 8-).

Reply:
I tried improving upon your suggestion :) See 5165321a4